packages/RichTextEditing-Core.package/DocumentReadStream.class/instance/text..st

```diff
@@ -1,4 +0,0 @@
-accessing
-text: aText
-
```

Comment:
We wanted to have a more self explanatory name than collection

Reply:
Not used, what is the purpose